foreman/data_refinery_foreman/settings.py

```diff
@@ -93,6 +93,10 @@ def get_env_variable(var_name):
         'PASSWORD': get_env_variable('DATABASE_PASSWORD'),
         'HOST': get_env_variable('DATABASE_HOST'),
         'PORT': get_env_variable('DATABASE_PORT'),
+
+        'OPTIONS': {
+            'connect_timeout': get_env_variable('DATABASE_TIMEOUT')
```

Comment:
I was running into an issue with my database timing out during development. While debugging that I changed the timeout to 5 seconds and decided that there wasn't really any time during development I'd want to wait 300 seconds to find out my database is timing out. So I made it an environment variable so I could have the short timeout during development..